packages/graphql/src/schema/resolvers/subscriptions/where/authorization.ts

```diff
@@ -60,3 +60,38 @@ export function subscriptionAuthorization({
 
     return multipleConditionsAggregationMap.OR(results);
 }
+
+function authorizationEventMatchesEvent(
+    authorizationEvent: "CREATED" | "UPDATED" | "DELETED" | "RELATIONSHIP_CREATED" | "RELATIONSHIP_DELETED",
+    event: "create" | "update" | "delete" | "create_relationship" | "delete_relationship"
+): boolean {
+    switch (authorizationEvent) {
+        case "CREATED":
+            if (event === "create") {
+                return true;
+            }
+            break;
```

Comment:
What about?

```suggestion
            return event === "create"
```

It will make the switch case less verbose and remove the `breaks`